File: V3/ServerApp/src/main/java/lk/earth/earthuniversity/report/entity/LandReport.java
package lk.earth.earthuniversity.report.entity;

import javax.persistence.Entity;
import javax.persistence.Id;

public class LandReport {

    private Integer id;
    private String landtype;
    private Long count;
    private double percentage;

    public LandReport() { }

    public LandReport(String landtype, Long count) {
        this.landtype = landtype;
        this.count = count;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getlandtype() {
        return landtype;
    }

    public void setlandtype(String landtype) {
        this.landtype = landtype;
    }

    public Long getCount() {
        return count;
    }

    public void setCount(Long count) {
        this.count = count;
    }

    public double getPercentage() {
        return percentage;
    }

    public void setPercentage(double percentage) {
        this.percentage = percentage;
    }
}

File: V3/ClientApp/src/app/report/entity/landreport.ts
export class landreport {
  public id!: number;
  public landtype!: string;
  public count!: number;
  public percentage!: number;

  constructor(id: number, landtype: string, count: number, percentage: number) {
    this.id = id;
    this.landtype = landtype;
    this.count = count;
    this.percentage = percentage;
  }
}
